windows.js

```diff
@@ -81,6 +83,8 @@ export async function install(platform, engine, version) {
     await installGCCTools(msys2Type)
   }
 
+  if (version === 'mswin') { await installVCPkg() }
```

Comment:
```suggestion
  if (version === 'mswin') {
    await installVCPkg() 
  }
```
So control flow can be seen visually without scanning the whole line

Reply:
Thanks, AFK for a bit.